### PR TITLE
Add blog post on FerretDB v0.92

### PR DESCRIPTION
**Pull request by Fashander**
# Description

## Readiness checklist

* [ ] I added/updated unit tests.
* [ ] I added/updated integration/compatibility tests.
* [ ] I added/updated comments and checked rendering.
* [ ] I made spot refactorings.
* [ ] I updated user documentation.
* [ ] I ran `task all`, and it passed.
* [x] I ensured that PR title is good enough for the changelog.
* [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Assignee, Labels, Project and project's Sprint fields.
* [x] I marked all done items in this checklist.

### CODE REVIEW

File: website/blog/2023-02-28-ferretdb-release-0-9-2.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+![FerretDB v0.9.1 - Minor Release](/img/blog/ferretdb_v0.9.2.jpg)
+
+<!--truncate-->
+
+It’s been an intense and thrilling process toward bringing you the open-source alternative to FerretDB.
+
+Before we go into all the exciting new features in this release, let’s recap FerretDB and what we’re striving to achieve.
+[FerretDB](https://www.ferretdb.io/) is an [open-source alternative to MongoDB](https://blog.ferretdb.io/5-database-alternatives-mongodb-2023/) that converts MongoDB drivers and protocols into SQL with a backend on PostgreSQL.
+
+While our aim is not to be compatible with all the features in MongoDB, we hope to provide all the necessary features you need for real-life use cases, and so far, we’ve added some pretty cool features.
```
Comment (ptrfarkas) [inline website/blog/2023-02-28-ferretdb-release-0-9-2.md:22]: We are gradually increasing compatibility with MongoDB, so FerretDB can act as a fully open source drop-in replacement for it. MongoDB compatibility includes tools, drivers and the entire MongoDB ecosystem. Be sure to try it out!

File: website/blog/2023-02-28-ferretdb-release-0-9-2.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+date: 2023-02-28
+---
+
+We are delighted to announce the release of FerretDB version 0.9.2, which includes Tigris authentation, support for `$listIndexes`, `$addtoSet`, and more.
+
+![FerretDB v0.9.1 - Minor Release](/img/blog/ferretdb_v0.9.2.jpg)
+
+<!--truncate-->
+
+It’s been an intense and thrilling process toward bringing you the open-source alternative to FerretDB.
```
Comment (ptrfarkas) [inline website/blog/2023-02-28-ferretdb-release-0-9-2.md:17]: The past few weeks were exciting: our cooperation with the open source community is stronger than ever. The feedback of developers and service providers makes us confident that we are on the right track with creating  truly open source alternative to MongoDB.

**Review by ptrfarkas — DISMISSED** [review]
Looks good!

File: website/blog/2023-02-28-ferretdb-release-0-9-2.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+<!--truncate-->
+
+The past few weeks were exciting: our cooperation with the open source community is stronger than ever.
+The feedback of developers and service providers makes us confident that we are on the right track with creating truly open source alternative to MongoDB.
+
+Before we go into all the exciting new features in this release, let’s recap FerretDB and what we’re striving to achieve.
+[FerretDB](https://www.ferretdb.io/) is an [open-source alternative to MongoDB](https://blog.ferretdb.io/5-database-alternatives-mongodb-2023/) that converts MongoDB drivers and protocols into SQL with a backend on PostgreSQL.
+
+We are gradually increasing compatibility with MongoDB– including tools, drivers, and the entire ecosystem, so FerretDB can act as a fully open source drop-in replacement for it.
```
Comment (b1ron) [inline website/blog/2023-02-28-ferretdb-release-0-9-2.md:23]: increasing compatibility with MongoDB - 

Space before hyphen

File: website/blog/2023-02-28-ferretdb-release-0-9-2.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+Be sure to try it out!
+
+Here are some of the notable changes in our previous releases:
+
+* Basic support for aggregation pipelines
+* More filtering query pushed to the backend for Tigris and PostgreSQL
```
Comment (b1ron) [inline website/blog/2023-02-28-ferretdb-release-0-9-2.md:29]: More filter queries pushed down to the backend

File: website/blog/2023-02-28-ferretdb-release-0-9-2.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+* Client TLS certificate validation
+* Support for raspberry pi
+* Basic telemetry service
+
+This above list is by no means exhaustive; it’s only a snippet of some of the recent additions.
+If you missed our previous releases, see the [complete list of all the updates](https://github.com/FerretDB/FerretDB/releases/) to FerretDB.
+We’ll be adding lots of new features as we approach the FerretDB GA.
+If you have a feature request,feedback, or discovered a bug, please don’t hesitate to [contact us](https://docs.ferretdb.io/#community).
```
Comment (b1ron) [inline website/blog/2023-02-28-ferretdb-release-0-9-2.md:37]: Add a space after "request,"

File: website/blog/2023-02-28-ferretdb-release-0-9-2.md [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+We’d love to hear from you!
+
+Here are the latest updates on this new version of FerretDB:
+
+## New Features
+
+Coming into this release is support for the `listIndexes` command.
+With this command, you can now access a collection’s indexes.
+Besides `listindexes` command, we’ve also added support for `$addToSet` update operators, meaning you can easily add elements to an array as long as the element does not previously exist.
```
Comment (b1ron) [inline website/blog/2023-02-28-ferretdb-release-0-9-2.md:46]: update  operator (singular)

File: website/blog/2023-02-28-ferretdb-release-0-9-2.md [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+Use the `$addtoSet` operator to update the array with more elements.
+
+```js
+db.states.updateOne(
+  { _id: 1 },
+  { $addToSet: { colors: "green" } }
+)
+```
+
+Another new feature implemented in this release is the support for $pullArrray.
```
Comment (b1ron) [inline website/blog/2023-02-28-ferretdb-release-0-9-2.md:66]: Did you mean $pullAll?

File: website/blog/2023-02-28-ferretdb-release-0-9-2.md [hunk 8]
```diff
@@ -0,0 +1,107 @@
...
+This features updates an array by removing all entries of the specified element.
```
Comment (b1ron) [inline website/blog/2023-02-28-ferretdb-release-0-9-2.md:67]: in the specified array

File: website/blog/2023-02-28-ferretdb-release-0-9-2.md [hunk 9]
```diff
@@ -0,0 +1,107 @@
...
+Suppose we have the above document and want to update it using the $pullAll operator:
+
+```js
+db.tested.updateOne(
+    { _id: 1 },
+    { $pullAll: { scores: [ 0, 3 ] } }
+)
+```
+
+The collection is subsequently purged of all existing instances of the elements in the specified array.
```
Comment (b1ron) [inline website/blog/2023-02-28-ferretdb-release-0-9-2.md:83]: The document is effected not the collection. Unless the multi option is specified. Plus updateOne will only update a single document.

**Review by b1ron — CHANGES_REQUESTED** [review]
Looks great! I added some suggestions.

**Review by b1ron — DISMISSED** [review]
Perfecto. With small tweak if needed.

**Review by b1ron — DISMISSED** [review]

**Review by ptrfarkas — APPROVED** [review]

File: website/blog/2023-02-28-ferretdb-release-0-9-2.md [hunk 10]
```diff
@@ -0,0 +1,107 @@
...
+title: "FerretDB v0.9.2 - Tigris Authentication support"
+slug: ferretdb-v-0-9-2
+authors: [alex]
+description: "We are delighted to announce the release of FerretDB version 0.9.2, which includes Tigris authentation, support for `$listIndexes`, `$addtoSet`, and more."
+image: /img/blog/ferretdb_v0.9.2.jpg
+tags: [release]
+date: 2023-02-28
+---
+
+We are delighted to announce the release of FerretDB version 0.9.2, which includes Tigris authentation, support for `$listIndexes`, `$addtoSet`, and more.
```
Comment (noisersup) [inline website/blog/2023-02-28-ferretdb-release-0-9-2.md:11]:
```suggestion
We are delighted to announce the release of FerretDB version 0.9.2, which includes Tigris authentication, support for `$listIndexes`, `$addtoSet`, and more.
```

Comment (AlekSi) [inline website/blog/2023-02-28-ferretdb-release-0-9-2.md]:
```suggestion
We are delighted to announce the release of FerretDB version 0.9.2, which includes Tigris authentation, support for `listIndexes`, `$addtoSet`, and more.
```

`listIndexes` is a command: https://www.mongodb.com/docs/manual/reference/command/listIndexes/
`$addtoSet` is an operator: https://www.mongodb.com/docs/manual/reference/operator/update/addToSet/

File: website/blog/2023-02-28-ferretdb-release-0-9-2.md [hunk 11]
```diff
@@ -0,0 +1,107 @@
...
+
+![FerretDB v0.9.1 - Minor Release](/img/blog/ferretdb_v0.9.2.jpg)
+
+<!--truncate-->
+
+The past few weeks were exciting: our cooperation with the open source community is stronger than ever.
+The feedback of developers and service providers makes us confident that we are on the right track with creating a truly open source alternative to MongoDB.
+
+Before we go into all the exciting new features in this release, let’s recap FerretDB and what we’re striving to achieve.
+[FerretDB](https://www.ferretdb.io/) is an [open-source alternative to MongoDB](https://blog.ferretdb.io/5-database-alternatives-mongodb-2023/) that converts MongoDB drivers and protocols into SQL with a backend on PostgreSQL and Tigris.
```
Comment (AlekSi) [inline website/blog/2023-02-28-ferretdb-release-0-9-2.md:21]: Tigris does not use SQL

File: website/blog/2023-02-28-ferretdb-release-0-9-2.md [hunk 12]
```diff
@@ -0,0 +1,107 @@
...
+If you have a feature request, feedback, or discovered a bug, please don’t hesitate to [contact us](https://docs.ferretdb.io/#community).
+We’d love to hear from you!
+
+Here are the latest updates on this new version of FerretDB:
+
+## New Features
+
+Coming into this release is support for the `listIndexes` command.
+With this command, you can now access a collection’s indexes.
+Besides `listindexes` command, we’ve also added support for `$addToSet` update operator, meaning you can easily add elements to an array as long as the element does not previously exist.
```
Comment (AlekSi) [inline website/blog/2023-02-28-ferretdb-release-0-9-2.md:46]:
```suggestion

Besides `listindexes` command, we’ve also added support for `$addToSet` update operator, meaning you can easily add elements to an array as long as the element does not previously exist.
```

We need a blank line there

**Review by noisersup — CHANGES_REQUESTED** [review]
One small typo, overall the content looks good!

File: website/blog/2023-02-28-ferretdb-release-0-9-2.md [hunk 13]
```diff
@@ -0,0 +1,107 @@
...
+
+```js
+db.states.insertOne({
+  _id: 1,
+  name: "California",
+  colors: ["blue", "gold", "red"]
+})
+```
+
+Use the `$addtoSet` operator to update the array with more elements.
```
Comment (AlekSi) [inline website/blog/2023-02-28-ferretdb-release-0-9-2.md:57]: That example does not show what is the difference from `$push`:
* https://www.mongodb.com/docs/manual/reference/operator/update/push/
* https://www.mongodb.com/docs/manual/reference/operator/update/addToSet/

File: website/blog/2023-02-28-ferretdb-release-0-9-2.md [hunk 14]
```diff
@@ -0,0 +1,107 @@
...
+db.tested.updateOne(
+    { _id: 1 },
+    { $pullAll: { scores: [ 0, 3 ] } }
+)
+```
+
+The document is subsequently purged of all instances of the specified values from the existing array.
+
+Another great addition to our release party is the implementation of authentication for Tigris.
+All the newly added features showcase FerretDB’s dedication to creating a database that’s compatible with a wide variety of use cases.
```
Comment (AlekSi) [inline website/blog/2023-02-28-ferretdb-release-0-9-2.md:86]: Blank line there too?

File: website/blog/2023-02-28-ferretdb-release-0-9-2.md [hunk 15]
```diff
@@ -0,0 +1,107 @@
...
+
+## Bug fixes
+
+In addition to the new releases, we’ve identified and fixed some bugs found in previous FerretDB releases.
+For example, we found that Java drivers require support for setting `capped:false` on the `create` command.
+See our release to find out more.
```
Comment (AlekSi) [inline website/blog/2023-02-28-ferretdb-release-0-9-2.md:92]: I think that should be "release notes" with a link like below

File: website/blog/2023-02-28-ferretdb-release-0-9-2.md [hunk 16]
```diff
@@ -0,0 +1,107 @@
...
+
+## Documentation
+
+An important update to our documentation is setting up analytics for our blog and documentation.
+Having analytics set up enables us to identify potential areas for improvement and deliver top-notch content and user experience that resonates with our users on our documentation and blog.
+
+In other changes, we’ve added more instructions explaining how evaluation and bitwise query operators work on FerretDB and making extra changes to the overall documentation content.
+To learn more about all the changes in the new release, please see [our release note](https://github.com/FerretDB/FerretDB/releases/tag/v0.9.2).
```
Comment (AlekSi) [inline website/blog/2023-02-28-ferretdb-release-0-9-2.md:100]:
```suggestion
To learn more about all the changes in the new release, please see [our release notes](https://github.com/FerretDB/FerretDB/releases/tag/v0.9.2).
```

**Review by noisersup — DISMISSED** [review]

**Review by noisersup — APPROVED** [review]

**Review by w84thesun — APPROVED** [review]

**Pull request merged**